### PR TITLE
[explore-v2] add config option for explore v2 beta users, and send through v2 path

### PR DESCRIPTION
**Pull request by ascott**
- add config option for explore v2 beta users
- if user_id is in beta list, send through v2 path

chef PR https://git.musta.ch/airbnb/chef/pull/13591

plz review @airbnb/superset-reviewers

### CODE REVIEW

**Comment by ascott** [comment]
@airbnb/superset-reviewers would love to get some :eyes: on this so we can start beta testing asap!

**Comment by ascott** [comment]
@airbnb/superset-reviewers ping

**Comment by vera-liu** [comment]
Not an expert on chef changes, but these changes LGTM!

File: superset/views.py [hunk 1]
```diff
@@ -1427,6 +1429,11 @@ def explore(self, datasource_type, datasource_id):
             return self.save_or_overwrite_slice(
                 request.args, slc, slice_add_perm, slice_edit_perm)
 
+        # find out if user is in explore v2 beta group
+        # and set flag `is_in_explore_v2_beta`
+        explore_v2_users = config.get('EXPLORE_V2_BETA_USERS', [])
```
Comment (bkyryliuk) [inline superset/views.py]: please add a test

**Comment by ascott** [comment]
@bkyryliuk changed this to use a role rather than ids from the config. PTAL!

File: tests/core_tests.py [hunk 2]
```diff
@@ -156,6 +156,24 @@ def test_slices(self):
             print("[{name}]/[{method}]: {url}".format(**locals()))
             self.client.get(url)
 
+    def test_slices_V2(self):
```
Comment (ascott) [inline tests/core_tests.py]: this test is passing, but for some reason `test_client (tests.druid_tests.DruidTests)` is now failing. need to investigate.

File: superset/views.py [hunk 3]
```diff
@@ -1484,6 +1486,13 @@ def explore(self, datasource_type, datasource_id):
             return self.save_or_overwrite_slice(
                 request.args, slc, slice_add_perm, slice_edit_perm)
 
+        # find out if user is in explore v2 beta group
+        # and set flag `is_in_explore_v2_beta`
+        is_in_explore_v2_beta = False
+        roles = (r.name for r in get_user_roles())
```
Comment (bkyryliuk) [inline superset/views.py]: s/roles = (r.name for r in get_user_roles())/roles = [r.name for r in get_user_roles()]
using list will be safer here

File: tests/core_tests.py (hunk 2 above)
Comment (ascott) [inline tests/core_tests.py]: i'll DM you, thx @bkyryliuk

**Pull request merged**